api_tests/entities/update_claims.test.coffee

```diff
@@ -66,3 +66,20 @@ describe 'entities:update-claims', ->
     .catch undesiredErr(done)
 
     return
+
+  it 'should accept rapid updates on the same entity', (done)->
+    authorsUris = [ 'wd:Q192214', 'wd:Q206685', 'wd:Q281411', 'wd:Q312835', 'wd:Q309945' ]
+    createWork()
+    .then (work)->
+      { uri: workUri } = work
+      Promise.all authorsUris.map((uri)-> updateClaim work._id, 'wdt:P50', null, uri)
```

Comment:
this line starts 5 `/api/entities?action=update-claim` requests at once, so that's some pretty quick editing ^^ if you revert the changes on the non-tests files, you should see this test pass to red due to CouchDB doc edit conflicts